### Business Statistics - Week 1 - Quiz

In [1]:
#import the important packages
import pandas as pd #library used for data manipulation and analysis
import numpy as np # library used for working with arrays.
import matplotlib.pyplot as plt # library for plots and visualisations
import seaborn as sns # library for visualisations
%matplotlib inline 

import scipy.stats as stats # this library contains a large number of probability distributions as well as 
# ...a growing library of statistical functions.

#### Below work (3 cells) is for question 9

In [5]:
lower=stats.norm.cdf(-0.8869,loc=0,scale=1)
lower

0.18756637083375943

In [6]:
upper=stats.norm.cdf(0.7652,loc=0,scale=1)
upper

0.7779237701569741

In [11]:
result=upper-lower
result

0.5903573993232147

## Questions

## Q 1

#### McDonald's is performing a marketing survey about the customer’s behavior when it comes to eating Burger. The company is trying to understand the favorite taste of their customers in order to redesign the menu. The company gathered the data of 1000 people of different ages, genders, and locations in order to understand the most desired taste of the entire population. Which of the following statistical methods can be used to get the inference about the population ( all customers )?

* The company is trying to know the characteristics of the population using sample statistics. Hence, Inferential Statistics should be used to draw an inference about the population.

## Q 2

#### Which of the following is not a property of a Normal Distribution?

* --> It is slightly skewed towards left
* It is bell-shaped
* It ranges from negative infinity to positive infinity
* Its central tendency measures (mean, median, mode) are equal

## Q 3

#### The World Economic Forum found that  12% of the U.S. population earn more than 2M per year.  Amazon decided to conduct an internal survey and chose 20 employees at random. Which of the following scipy functions will be used to find the probability that exactly 5 of the 20 employees earn more than 2M per year?

* stats.binom.pmf()

>* X is the Binomial Random variable which represents the number of employees. We are interested in finding the P(X=5).
>* pmf() function is used to find the absolute probability of k successes in n number of trials where the probability of success for each trial is p
>* the binomial distribution with parameters n and p is the discrete probability distribution of the number of successes in a sequence of n independent experiments, each asking a yes–no question, and each with its own Boolean-valued outcome: success or failure.

## Q 4

#### Which one of these random variables is continuous?

* --> The time taken by a randomly selected learner to complete the quiz
* The number of two wheelers, a randomly selected person has.
* The number of men taller than 68 inches in a random sample of 5 men.
* The number of phone calls received in a call center

>* Time taken by a randomly selected learner in a quiz is continuous because it can take any value within a given range

## Q 5

#### Suppose X is a continuous random variable that is uniformly distributed between 3 and 8. Which one of the following functions can be used to find P(X<=5.8)?

* import scipy.stats as st
* st.uniform.cdf(5.8,loc=3,scale=5)

>* A continuous random variable X is uniformly distributed between 3 and 8. CDF is used to find the probability that X will take a value less than or equal to x. scipy.stats.norm has a function called cdf() which is used to find the P(X<=x)

## Q 6

#### Scientists want to know the incubation period of coronavirus. They have collected the blood samples of 100 people and found that the mean incubation period is 43 hours with a standard deviation of 3.6 hours. Which of the following code is correct to calculate the 95% confidence interval of the mean Incubation period.

* from scipy import stats
* import numpy as np
* confidence_coef = .95
* stats.t.interval(confidence_coef, df = 100-1, loc =43 , scale = 3.6/np.sqrt(100))

> Here, population standard deviation is not known therefore we need to use t-distribution to calculate a confidence interval. stats.t.interval() function is used to calculate the interval which takes confidence level, degree of freedom, sample mean ( loc ), and standard error ( scale ) as the arguments.

## Q 7

#### Suppose that the heights of men in the United States are normally distributed with a mean of 65.6 inches and a standard deviation of 2.7 inches. Which of the following functions can be used to find the probability that the mean height is greater than 63.0 inches?

* import scipy.stats as st
* 1-st.norm.cdf(63,loc=65.6,scale=2.7)

> Use 1 - cdf() when greater than...
>* CDF is used to find the probability that X ( random variable ) will take a value less than or equal to x ( P( X<=x). But here we are interested in P(X>63). It can be calculated as follows -- P(X>63) = 1-P(X<=63)

## Q 8

#### In which of the following situations can a Uniform Distribution be applied? [B, C, D]

* A. When each value of a continuous random variable is not equally likely to occur.
* --> B. When each discrete value is equally likely to occur.
* --> C. When each value of a continuous random variable is equally likely to occur
* --> D. Guessing the birthday of a person

> In uniform distribution, all outcomes (each value of the random variable) have an equal probability of occurrence. The birthday of a person can be on any date of the year. So, the probability of each date is equal, that is 1/365 (for a year with 365 days)

## Q 9

#### A college conducts a common test for all the students. For the Mathematics portion of this test, the scores are normally distributed with a mean of 502 and a standard deviation of 115. Based on this information, what percentage of students would be expected to score between 400 and 590?

* First calculate Z-score for 400 --> (400-502)/115 = -0.8869
* Then calculate Z-score for 590 --> (590-502)/115 = 0.7652
* Proportion for Z= -0.8869 -- stats.norm.cdf(-0.8869,loc=0,scale=1) = 0.1875 # scalse is 0 to 1
* Proportion for Z=0.7652 -- stats.norm.cdf(0.7652,loc=0,scale=1) = 0.77792 # scalse is 0 to 1
* Then subtract: 0.77792-0.1875 = 0.5942 ~ 59.42%

> Neither of the available answers were correct in the test.

## Q 10

#### At Intel, the sizes of processor chips have a mean of 1 cm with a standard deviation of 0.2 cm. Multiple random samples, each consisting of 35 processor chips, are taken. The sample mean and standard deviation for each sample are recorded to form a sampling distribution. What is the mean and standard deviation of this sampling distribution?

* According to the concept of the sampling distribution of means and the Central Limit theorem, X will follow a normal distribution with a mean of 1 cm and a standard deviation of 0.2/sqrt(35).
* So, mean = 1 cm
* and, standard deviation = 0.2/sqrt(35) = 0.0338
